src/features/profile/Profile.tsx

```diff
@@ -0,0 +1,94 @@
+import { useCallback, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import { useSelector } from "react-redux";
+import useProfileState from "./use-profile-state";
+import { AuthState } from "features/auth/store/types";
+import { fetchOneProfile } from "./services/profile.service";
+import ShowSkills from "./components/show-skills";
+import ShowExperiences from "./components/show-experiences";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import IconButton from "@mui/material/IconButton";
+import EditIcon from "@mui/icons-material/Edit";
+
+const Profile = () => {
+  const { profile, setProfile } = useProfileState();
+
+  const id = parseInt(useParams<RouteParams>().id!);
+
+  function useAuthState(): AuthState {
+    return useSelector<{ auth: AuthState }>((state) => state.auth) as AuthState;
+  }
+  const { token } = useAuthState();
+  const loggedUser = useAuthState().user;
```

Comment:
```suggestion
  const { token, user } = useAuthState();
```
O... si quieres usar el alias
```suggestion
  const { token, user: loggedUser } = useAuthState();
```

Reply:
curioso, no sabia que se podia hacer eso, gracias xD